src/index.js

```diff
@@ -105,31 +104,27 @@ function ncc (
     existingAssetNames.push(`${filename}.cache`);
     existingAssetNames.push(`${filename}.cache${ext}`);
   }
-  const resolvePlugins = [];
-  // add TsconfigPathsPlugin to support `paths` resolution in tsconfig
-  // we need to catch here because the plugin will
-  // error if there's no tsconfig in the working directory
-  let fullTsconfig = {};
-  try {
-    const configFileAbsolutePath = walkParentDirs({
-      base: process.cwd(),
-      start: dirname(entry),
-      filename: 'tsconfig.json',
-    });
-    fullTsconfig = loadTsconfig(configFileAbsolutePath) || {
-      compilerOptions: {}
-    };
 
-    const tsconfigPathsOptions = { silent: true }
-    if (fullTsconfig.compilerOptions.allowJs) {
-      tsconfigPathsOptions.extensions = SUPPORTED_EXTENSIONS
-    }
-    resolvePlugins.push(new TsconfigPathsPlugin(tsconfigPathsOptions));
+  let tsconfig = {};
+  try {
+    const configPath = join(process.cwd(), 'tsconfig.json');
+    const contents = fs.readFileSync(configPath, 'utf8')
```

Comment:
How about using ts api to parse the config? It has type hint

```
const config = ts.readConfigFile(tsconfigPath, ts.sys.readFile).config;
const compilerOptions = ts.parseJsonConfigFileContent(
        config,
        ts.sys,
        './',
).options;
```

Reply:
Sounds good. I'll probably go with the other PR instead https://github.com/vercel/ncc/pull/777